pipelines/internal/commands/run/run.go

```diff
@@ -162,6 +162,7 @@ var (
 	gpuType        = flags.String("gpu-type", "nvidia-tesla-k80", "the GPU type to attach")
 	command        = flags.String("command", "", "a single command line to execute")
 	fuse           = flags.Bool("fuse", false, "if true, use FUSE to localize inputs (see README)")
+	debug          = flag.Bool("debug", false, "if true, an shh server will be started")
```

Comment:
let's make this flag called 'ssh' instead (we may want to use --debug for other things)